Utility/integration-test.sh

```diff
@@ -4,11 +4,14 @@
 JSON_FILES=`ls -d Examples/PDK/*.json`
 
 # Generate Objective-C models
-.build/debug/plank  --output_dir=Examples/Cocoa/Sources/objc/ $JSON_FILES
+.build/debug/plank  --output_dir=Examples/Cocoa/Objc/Sources/objc $JSON_FILES
 
 # Generate flow types for models
 .build/debug/plank --lang flow  --output_dir=Examples/JS/flow/ $JSON_FILES
 
+# Generate Swift models
+.build/debug/plank --lang swift  --output_dir=Examples/Cocoa/Swift/Sources/swifty $JSON_FILES
```

Comment:
This doesn't have to be in this commit but it might be worthwhile to just have ObjC and Swift at the top level rather than under Cocoa.